uber/models.py

```diff
@@ -1122,15 +1122,15 @@ def floating(self):
 
     @property
     def new_ribbon(self):
-        return c.DEALER_RIBBON if self.is_dealer else c.NO_RIBBON
+        return c.DEALER_RIBBON if self.is_dealer else ''
```

Comment:
Functionally speaking, adding new badges to a group (which the `new_ribbon` property is used for) is the main thing that didn't get 'upgraded' to use multichoice options. It's much easier this way, and the use-case of creating blank badges with multiple ribbons is pretty rare.